setup.cfg

```diff
@@ -32,7 +32,7 @@ package_dir =
 packages = find:
 python_requires = >=3.8
 install_requires =
-    torch>=1.12
+    torch==2.2.2
```

Comment:
Thanks for the contribution! The requirement seems to be overly restrictive though (it will force users with lower versions of PyTorch to update), can you change it to inequality constraints?

Reply:
Thank you, that's right! Updated PyTorch version constraint in the cfg to torch>=1.12,<2.3.0 to have compatibility with earlier versions while avoiding the issue introduced in 2.3.0. Please review the updated pull request.